lightning/src/chain/onchaintx.rs

```diff
@@ -623,19 +624,31 @@ impl<ChannelSigner: WriteableEcdsaChannelSigner> OnchainTxHandler<ChannelSigner>
 			return inputs.find_map(|input| match input {
 				// Commitment inputs with anchors support are the only untractable inputs supported
 				// thus far that require external funding.
-				PackageSolvingData::HolderFundingOutput(..) => {
+				PackageSolvingData::HolderFundingOutput(output) => {
 					debug_assert_eq!(tx.txid(), self.holder_commitment.trust().txid(),
 						"Holder commitment transaction mismatch");
+
+					let conf_target = ConfirmationTarget::HighPriority;
+					let package_target_feerate_sat_per_1000_weight = cached_request
+						.compute_package_feerate(fee_estimator, conf_target, force_feerate_bump);
+					if let Some(input_amount_sat) = output.funding_amount {
```

Comment:
Code can be better documented like `package_target_feerate_sat_per_1000_weight` as technically there is no “package” yet here we have no evaluate if the commitment needs a CPFP and therefore become a package. 

new name could be “commitment_alone_target_feerate_sat_per_1000_weight`, somehow the `compute_package_feerate` coming from my old refactor sorry for the confusion.